common/include/pcl/experimental/executor/best_fit.hpp

```diff
@@ -73,15 +81,21 @@ struct executor_runtime_checks {
 namespace detail {
 
 template <typename Function, typename Executor, typename = void>
-bool execute(Function&&, Executor&) {
+bool
+execute(Function&&, Executor&)
+{
   return false;
 }
 
-template <typename Function, template <typename...> class Executor,
-          typename... Properties,
-          typename std::enable_if<executor::is_executor_available_v<Executor>,
-                                  int>::type = 0>
-bool execute(Function&& f, Executor<Properties...>& exec) {
+template <
+    typename Function,
+    template <typename...>
+    class Executor,
```

Comment:
Breaking clang-format 101 :D

Reply:
Fixed it manually. Should I disable clang-format for that section of code?